### PR TITLE
added react-toastify for showing errors

### PR DESCRIPTION
**Pull request by mngagan**
showing toastify messages when getting any error from services

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -411,7 +413,7 @@ export default function Main() {
         </div>
         <div className='results-container'>
           {githubContext.loading && <div className='loading' />}
-          {githubContext.errorMessage.length > 0 && (
+          {false && githubContext.errorMessage.length > 0 && ( // fix for issue 51 https://github.com/jeffslofish/open-source-help-wanted/issues/51
```
Comment (jeffslofish) [inline src/components/Main.js:416]: Can you just remove all this code that never executes? (Lines 416-418)

**Review by jeffslofish — CHANGES_REQUESTED** [review]
Thanks for this! Can you just address the one comment I made? Thanks!

**Comment by mngagan** [comment]
> done

**Comment by jeffslofish** [comment]
Thanks! 👍

**Pull request merged**